pkg/api/http/consts/consts.go

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright 2025 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package consts
+
+const DaprAppIDHeader = "Dapr-App-Id"
```

Comment:
Was it intentional to uppercase each word here? I see the const before was:
```suggestion
const DaprAppIDHeader = ""dapr-app-id""
```

Reply:
Yes, HTTP headers, although can be case insensitive, Go uses [CanonicalMIMEHeaderKey](https://pkg.go.dev/net/textproto#CanonicalMIMEHeaderKey) to convert to Pascal-Case when using `Get` or `Set` on headers.